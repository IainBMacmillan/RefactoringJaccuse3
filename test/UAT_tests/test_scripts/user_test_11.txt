Enter
A
T
Enter
B
1
Enter
T
Enter
Q