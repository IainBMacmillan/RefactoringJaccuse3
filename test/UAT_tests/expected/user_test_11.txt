J'ACCUSE! (a mystery game)")
    By Al Sweigart al@inventwithpython.com 
    Inspired by Homestar Runner's "Where's an Egg?" game 

    You are the world-famous detective, Mathilde Camus. 
    ZOPHIE THE CAT has gone missing, and you must sift through the clues. 
    Suspects either always tell lies, or always tell the truth. Ask them 
    about other people, places, and items to see if the details they give are 
    truthful and consistent with your observations. Then you will know if 
    their clue about ZOPHIE THE CAT is true or not. Will you find ZOPHIE THE 
    CAT in time and accuse the guilty party? 
    
Press Enter to begin...
Time left for patch testing
 You are in your TAXI. Where do you want to go?
(Q)UIT GAME
> 
Time left for patch testing
 You are at the ALBINO ALLIGATOR PIT.
 BILL MONOPOLIS with the 5 DOLLAR GIFT CARD is here.

 (J) "J'ACCUSE!" (3 accusations left)
 (Z) Ask if they know where ZOPHIE THE CAT is.
 (T) Go back to the TAXI.
 (1) Ask about BILL MONOPOLIS
 (2) Ask about 5 DOLLAR GIFT CARD
> Press Enter to continue...
Time left for patch testing
 You are in your TAXI. Where do you want to go?
(A)LBINO ALLIGATOR PIT (bill monopolis, 5 dollar gift card)
(Q)UIT GAME
> 
Time left for patch testing
 You are at the BOWLING ALLEY.
 CECIL EDGAR VANDERTON with the ANIME VHS TAPE is here.

 (J) "J'ACCUSE!" (3 accusations left)
 (Z) Ask if they know where ZOPHIE THE CAT is.
 (T) Go back to the TAXI.
 (1) Ask about BILL MONOPOLIS
 (2) Ask about 5 DOLLAR GIFT CARD
 (3) Ask about CECIL EDGAR VANDERTON
 (4) Ask about ANIME VHS TAPE
> They give you this clue: "ALBINO ALLIGATOR PIT"
Press Enter to continue...
Time left for patch testing
 You are at the BOWLING ALLEY.
 CECIL EDGAR VANDERTON with the ANIME VHS TAPE is here.

 (J) "J'ACCUSE!" (3 accusations left)
 (Z) Ask if they know where ZOPHIE THE CAT is.
 (T) Go back to the TAXI.
 (1) Ask about BILL MONOPOLIS
 (2) Ask about 5 DOLLAR GIFT CARD
 (3) Ask about CECIL EDGAR VANDERTON
 (4) Ask about ANIME VHS TAPE
> Press Enter to continue...
Time left for patch testing
 You are in your TAXI. Where do you want to go?
(A)LBINO ALLIGATOR PIT (bill monopolis, 5 dollar gift card)
(B)OWLING ALLEY        (cecil edgar vanderton, anime vhs tape)
(Q)UIT GAME
> Thanks for playing!
